copy(fxa-auth-server): update automatedEmail footer and

millsoper opened the pull request:
passwordResetAccountRecovery email
Because:

* We want to update the copy for the passwordResetAccountRecovery email, and also want to completely update the automatedEmailChangePassword footer.

This commit:

* Updates the footer and the email copy

Closes # https://mozilla-hub.atlassian.net/browse/FXA-5193

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)
<img width="1129" alt="Screen Shot 2022-09-08 at 4 45 10 PM" src="https://user-images.githubusercontent.com/11150372/189244574-a81e3a31-f27e-40f7-bef4-98cb6fef4ed8.png">

bcolsson commented on packages/fxa-auth-server/lib/senders/emails/templates/passwordResetAccountRecovery/en.ftl:
```diff
@@ -1,7 +1,8 @@
+passwordResetAccountRecovery-subject-2 = Your password has been reset
+passwordResetAccountRecovery-title-2 = Password reset successfully
+passwordResetAccountRecovery-description-2 = You used your account recovery key to update your password from:
+passwordResetAccountRecovery-action-2 = Create a new account recovery code
```
```suggestion
# Text for button action to initiate creating new account recovery code
passwordResetAccountRecovery-action-2 = Create a new account recovery code
```
Suggest adding comment for context to localize.

dschom replied:
Should this be 'account recovery key' per FXA-4423?

bcolsson commented on packages/fxa-auth-server/lib/senders/emails/templates/passwordResetAccountRecovery/en.ftl:
```diff
@@ -1,7 +1,8 @@
+passwordResetAccountRecovery-title-2 = Password reset successfully
+passwordResetAccountRecovery-description-2 = You used your account recovery key to update your password from:
+passwordResetAccountRecovery-action-2 = Create a new account recovery code
+passwordResetAccountRecovery-regen-required-base = You’ll need to sign in again on all of your synced devices. Remember to create a new account recovery key to replace the one you used
```
I think it'd be more straightforward for localizers to have this text in the mjml and txt strings, rather than calling the -base string.

dschom commented on packages/fxa-auth-server/lib/senders/emails/partials/appBadges/en.ftl:
```diff
@@ -4,5 +4,18 @@ body-android-badge = <img data-l10n-name="google-play-badge" alt="Download { $pr
+ios-download-plaintext = Get { $productName } on the App Store:
+# Variables:
+#  $productName (String) - The name of the product to be downloaded, e.g. Mozilla VPN, or Firefox
+another-device-plaintext = Get { $productName } on another device:
```
Perhaps test coverage on these new strings should be added? Note, there is currently test coverage on `body-ios-badge` and  `body-android-badge` l10n strings.

dschom commented on packages/fxa-auth-server/lib/senders/emails/templates/passwordResetAccountRecovery/includes.json:
```diff
@@ -1,10 +1,10 @@
-    "id": "passwordResetAccountRecovery-action",
-    "message": "Create new recovery key"
+    "id": "passwordResetAccountRecovery-action-2",
+    "message": "Create a new account recovery code"
```
Should this be 'account recovery key' per FXA-4423?

dschom commented on packages/fxa-auth-server/lib/senders/emails/partials/automatedEmailChangePassword/en.ftl:
```diff
@@ -1,5 +1,6 @@
-# Variables:
-#  $passwordChangeLink (String) - Link to https://accounts.firefox.com/settings/change_password
-automated-email-change-plaintext = This is an automated email; if you didn’t add a new device to your { -product-firefox-account }, you should change your password immediately at { $passwordChangeLink }
+automated-email-change-2 = If you didn’t take this action, <a data-l10n-name="passwordChangeLink">change your password</a> right away.
```
Perhaps test coverage should be added on this string?

dschom commented on packages/fxa-auth-server/lib/senders/emails/templates/passwordResetAccountRecovery/index.mjml:
```diff
@@ -19,14 +19,15 @@
-  buttonL10nId: "passwordResetAccountRecovery-action",
-  buttonText: "Create new recovery key"
+  buttonL10nId: "passwordResetAccountRecovery-action-2",
+  buttonText: "Create a new account recovery code"
```
Should this be 'account recovery key'?

dschom commented on packages/fxa-auth-server/lib/senders/emails/partials/appBadges/en.ftl:
```diff
@@ -4,5 +4,18 @@ body-android-badge = <img data-l10n-name="google-play-badge" alt="Download { $pr
+another-desktop-device-2 = Or, install { $productName } on <a data-l10n-name="anotherDeviceLink">another desktop device</a>.
+# Variables:
+#  $productName (String) - The name of the product to be downloaded, e.g. Mozilla VPN, or Firefox
+another-device-2 = Or, install { $productName } on <a data-l10n-name="anotherDeviceLink">another device</a>.
```
Per the [doc section 1.2](https://docs.google.com/document/d/18gF_7Ux-jE_qfEFciOcPml0-G1SfHkjiJSMPwh2hnSc/edit#heading=h.q439jqhxcydz), I think this is now suppsoed to say `Install`, not `Or, install`.

dschom commented on packages/fxa-auth-server/lib/senders/emails/partials/appBadges/index.txt:
```diff
@@ -0,0 +1,8 @@
+android-download-plaintext = "Get <%- locals.productName %> on Google Play:"
+<%- locals.androidUrl %>
+
+ios-download-plaintext = "Get <%- locals.productName  %> on the App Store:"
```
Per the [doc](https://docs.google.com/document/d/18gF_7Ux-jE_qfEFciOcPml0-G1SfHkjiJSMPwh2hnSc/edit#heading=h.q439jqhxcydz) App Store should be first.

dschom commented on packages/fxa-auth-server/lib/senders/emails/partials/appBadges/index.txt:
```diff
@@ -0,0 +1,8 @@
+ios-download-plaintext = "Get <%- locals.productName  %> on the App Store:"
+<%- locals.iosUrl %>
+
+another-device-plaintext = "Get <%- locals.productName %> on another device:"
```
Per the [doc](https://docs.google.com/document/d/18gF_7Ux-jE_qfEFciOcPml0-G1SfHkjiJSMPwh2hnSc/edit#heading=h.q439jqhxcydz) This should say 'Install Firefox' not 'Get Firefox'

dschom reviewed (changes requested):
@millsoper I notice a couple things that need to be updated:

- A few wording descrepancies between what's here and what's in the doc.
- Not sure why tests are failing
- A couple story book templates fail as well, e.g. checkout cadReminderFirst.

<img width="927" alt="image" src="https://user-images.githubusercontent.com/94418270/189461401-a0b1ade7-9610-4c45-98b0-d3deb654c37f.png">

millsoper commented:
> @millsoper I notice a couple things that need to be updated:
> 
>     * A few wording descrepancies between what's here and what's in the [doc](https://docs.google.com/document/d/18gF_7Ux-jE_qfEFciOcPml0-G1SfHkjiJSMPwh2hnSc/edit#heading=h.q439jqhxcydz).
> 
>     * Not sure why tests are failing
> 
>     * A couple story book templates fail as well, e.g. checkout cadReminderFirst.
> 
> 
> <img alt="image" width="927" src="https://user-images.githubusercontent.com/94418270/189461401-a0b1ade7-9610-4c45-98b0-d3deb654c37f.png">

Thank you so much Dan!! I added tests, got the broken test fixed and addressed the copy issues -- I think this should work now

dschom commented:
@millsoper Looks good to me! Thanks for the updates.

millsoper commented:
> @millsoper Alright, things are looking pretty good. I noticed that the 'Install Firefox' section _(aka /partials/appBadges/index.txt)_ is now missing from the text template in cadReminderFirst/Second templates. Other than that I think I would r+ now.

....hm. I'll take a look and fix it!

dschom commented:
@millsoper Ah, good point, I can r+ then. Perhaps another issue should be filed for this if one doesn't already exist?
